packages/angular/ssr/src/handler.ts

```diff
@@ -16,6 +16,8 @@
  */
 export type RequestHandlerFunction = (
   request: Request,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ...ignoredArgs: any[]
```

Comment:
Instead of using `any` here, can we instead use generics to infer the type? 

Example

```ts
export type RequestHandlerFunction<T = unknown> = (
  request: Request,
  requestContext?: T,
) => Promise<Response | null> | null | Response;

export function createRequestHandler<T = unknown>(
  handler: RequestHandlerFunction<T>,
): RequestHandlerFunction<T> {

}
```